.babelrc

```diff
@@ -5,7 +5,10 @@
     "add-module-exports",
     "transform-decorators-legacy",
     "transform-class-properties",
-    "transform-react-display-name"
```

Comment:
A note: I think dropped dependencies are not removed from yarn.lock automatically. But we can worry about that when making a release